css/style.css

```diff
@@ -1157,6 +1157,7 @@ ul {
   left: 0;
   mix-blend-mode: multiply;
   position: fixed;
+  backdrop-filter: blur(10px);
 }
 
 .mobile-menu-navbar {
```

Comment:
- [ ] The hamburger button and the text **My Logo** are hidden by bg images and text respectively. To verify this, let's kindly check the screenshots below: That of Figma and yours on a tabular form. 

| Yours | Figma |
| --- | --- |
| ![Screenshot from 2022-12-19 21-20-39](https://user-images.githubusercontent.com/63087491/208516315-81f73033-ed90-4d61-8305-ecad480b782c.png) |![Screenshot from 2022-12-19 21-26-04](https://user-images.githubusercontent.com/63087491/208516357-eddba3b3-1449-4e52-9288-44b058d4920f.png) |


- Hence, I couldn't click on the hamburger button to check the other functionalities.